### PR TITLE
Add `SECURITY_CONTACTS`

### PR DESCRIPTION
**Pull request by justaugustus**
Closes: #19 
/sig azure

Signed-off-by: Stephen Augustus <foo@agst.us>

### CODE REVIEW

**Comment by colemickens** [comment]
Looks fine to me but I'll wait for @khenidak in case there are more Azure folks to include.

File: SECURITY_CONTACTS [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# [Embargo Policy](https://github.com/kubernetes/sig-release/blob/master/security-release-process-documentation/security-release-process.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+colemickens
+justaugustus
+khenidak
+shubheksha
```
Comment (feiskyer) [inline SECURITY_CONTACTS]: @justaugustus Could you also include me in the list?

Comment (andyzhangx) [reply SECURITY_CONTACTS]: @justaugustus also include me: andyzhangx, thanks.

**Comment by justaugustus** [comment]
Did this get stuck?
/retest

**Pull request merged**